### ISSUE TITLE
Support for WP 5.0

### ISSUE THREAD

**Issue by striebwj** [ClassicPress/ClassicPress-Migration-Plugin#44]
Migration support for WP 5.0-RC1-43944, as discussed [here](https://forums.classicpress.net/t/plugin-unable-to-migrate-from-wp-5/185/1).

**Comment by nylen** [comment]
Things to look out for here:

- do new files in WP 5.0 get deleted with the migration?
- what happens to post content written or edited with Gutenberg? We probably need to at least add a warning that not all content may be editable after the migration.

**Comment by nylen** [comment]
Another one:  does WP 5.0 contain database changes as compared to 4.9.8?

**Comment by GTMAN** [comment]
/wp-admin/includes/update-core.php needs to be updated to include WP5 files in the $_old_files array. See attached a list of orphaned files:
[$_old_files.txt](https://github.com/ClassicPress/ClassicPress-Migration-Plugin/files/2660064/_old_files.txt)

**Comment by GTMAN** [comment]
/wp-admin/includes/update-core.php Line 20 needs to be updated to include WP5 files in the $_old_files array. See attached a list of orphaned files:
[$_old_files.txt](https://github.com/ClassicPress/ClassicPress-Migration-Plugin/files/2660068/_old_files.txt)

**Comment by nylen** [comment]
> does WP 5.0 contain database changes as compared to 4.9.8?

It does, so after migrating from WP 5.0 you see a "database upgrade" screen (which I guess is really a downgrade).

Support for migrating from WP 5.0 was added in commit b0a4a99 and version `0.4.0` of the plugin.

The remaining task (ensure that left-over WP 5.0 files are removed after migration) needs to be done in the ClassicPress code.  I've opened https://github.com/ClassicPress/ClassicPress/issues/286 to track that, so this issue can be closed now.
